androidsdk/src/main/java/com/wootric/androidsdk/objects/Settings.java

```diff
@@ -221,7 +247,7 @@ public void setFirstSurveyDelay(long firstSurvey) {
     }
 
     public Long getFirstSurveyDelay() {
-        return firstSurvey;
+        return firstSurvey == -1L? null : firstSurvey;
```

Comment:
@hachibu and because of this, we were always sending `first_survey_delay=31` to eligibility, which was wrong.